### PR TITLE
Feature dapps card carousel #131

### LINKED ISSUE #123: [BUG] navbar and footer bug in mobile view
### Description

image

image

### Expected behavior

I provided info. on what to change, please see the image

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #131: [FEATURE] Replace the dapps design to cards slider
### Detailed description

We should change the dapps section design

### Current

image

### Context

We think this feature is good 
https://tailwindui.com/components/application-ui/lists/grid-lists#component-5d72b187beb968e12db89fe49f272b18

### Possible implementation

_No response_

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by AbhinavMV**
#### Description of the change

closes #131 
closes #123 (This issue is coming again have reduced the text size in hero section)
#### Screenshots
For card carousel
In Light Mode:
image

In Dark Mode:
image

#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/151"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Awesome! 🚀 Thanks! 😄👍

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Can u put title in the center:

Dapps?
image

**Comment by AbhinavMV** [comment]
> Awesome! 🚀 Thanks! 😄👍

Thanks 😄
Actually i have a question 
i have found another bug in website **unrelated** to this card carousel. Should i raise an issue first and create another PR or fix it in this PR itself?

**Comment by AbhinavMV** [comment]
> Can u put title:
> 
> Dapps? image

Sorry missed it. Will add it now 👍

**Comment by vinzvinci** [comment]
> > Awesome! 🚀 Thanks! 😄👍
> 
> Thanks 😄 Actually i have a question i have found another bug in website **unrelated** to this card carousel. Should i raise an issue first and create another PR or fix it in this PR itself?

You can raise issue first then PR :)

**Comment by shuklaritvik06** [comment]
> > Awesome! 🚀 Thanks! 😄👍
> 
> Thanks 😄
> Actually i have a question 
> i have found another bug in website **unrelated** to this card carousel. Should i raise an issue first and create another PR or fix it in this PR itself?

I think it'll be good if you can open a new issue and then mention that issue in your PR description and solve the bug in this PR

**Comment by shuklaritvik06** [comment]
> > > Awesome! 🚀 Thanks! 😄👍
> > 
> > Thanks 😄
> > Actually i have a question 
> > i have found another bug in website **unrelated** to this card carousel. Should i raise an issue first and create another PR or fix it in this PR itself?
> 
> I think it'll be good if you can open a new issue and then mention that issue in this PR description and solve the bug in this PR

**Comment by AbhinavMV** [comment]
Fixed Dapps heading . 👍
Ok will raise an issue then.

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Sorry for another request.

Can u use our default font in Dapps? same with this font 
image

**Comment by AbhinavMV** [comment]
> Sorry for another request.
> 
> Can u use our default font in Dapps? same with this font image

Fixed it. 👍

**Review by vinzvinci — APPROVED** [review]
LGTM 👍 Thank you so much for your contributions 🚀

**Comment by AbhinavMV** [comment]
> LGTM 👍 Thank you so much for your contributions 🚀

Learning a lot by contributing. Thankyou 😄

File: src/pages/index.astro [hunk 1]
```diff
@@ -8,9 +8,10 @@ import DevToken from '@components/Home/Heros/DevToken.astro'
 
 import Backers from '@components/Home/Sections/Backers.astro'
 import Blog from '@components/Home/Sections/Blog.astro'
-import Dapps from '@components/Home/Sections/Dapps.astro'
+// import Dapps from '@components/Home/Sections/Dapps.astro'
```
Comment (krishguptadev) [inline src/pages/index.astro:11]: Humm this can be removed.

File: src/components/Home/Sections/Dapps.vue [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+});
+</script>
+<style>
+.carousel__slide {
+  padding: 5px;
+}
+
+.carousel__prev,
+.carousel__next {
+  width: 4rem !important;
```
Comment (krishguptadev) [inline src/components/Home/Sections/Dapps.vue:134]: Can't this be achieved using TailwindCSS

File: src/pages/index.astro (hunk 1 above)
Comment (AbhinavMV) [reply src/pages/index.astro:11]: hmm true. sorry forgot to do that 😅

File: src/components/Home/Sections/Dapps.vue (hunk 2 above)
Comment (AbhinavMV) [reply src/components/Home/Sections/Dapps.vue:134]: yes i tried to do that in tailwind. But don't know why it was not working. Maybe some bug in the vue3-carousel package.
Could you try doing that in tailwind?

File: src/pages/index.astro (hunk 1 above)
Comment (Panquesito7) [reply src/pages/index.astro:11]: Removed in https://github.com/web3community/devprotocol.xyz/commit/dd7e5d0e2b8289244a17ac348d043477a8897627

**Review by mkubdev — COMMENTED** [review]
Noice 👀

**Pull request merged**